enhancements/builds/entitlements.md

```diff
@@ -276,8 +209,7 @@ in a pod:
  - There are CSI volume and persistent volume types in k8s.
  - per [k8s volume docs](https://kubernetes.io/docs/concepts/storage/volumes/#csi) the CSI types do not support direct
  reference from a Pod and may only be referenced in a Pod via a `PersistentVolumeClaim`
- - To that end, one of the sidecar containers, the [external provisioner](https://kubernetes-csi.github.io/docs/external-provisioner.html),
- helps with this
+ - To that end, one of the sidecar containers, the [external provisioner](https://kubernetes-csi.github.io/docs/external-provisioner.html), helps with this
```

Comment:
Similar to your suggestion in https://github.com/openshift/cluster-samples-operator/pull/249, where we add a ref to this proposal and delete any repeat information, we should probably remove all these CSI factoids and simply add a ref to https://github.com/openshift/cluster-samples-operator/pull/249, no?

Reply:
yeah probably, can i leave that as a cleanup for you after you merge this?  you have a better understanding of CSI than me at this point and are probably better positioned to decide what minimal info needs to be provided here, vs what to defer to the lower level CSI EP.

Reply:
yep that works for me